index.html

```diff
@@ -495,6 +495,462 @@ <h2>
+          A user agent can use these members to associate the web application
+          with [=file type=] on the operating system.
+        </p>
+        <p>
+          A <dfn>file type</dfn> can be defined by a [=mime type=] and/or
+          [=file extension=]. A file belongs to a file type if the OS
+          determines it to have a mime type and/or its name ends with a certain
+          [=file extension=]. A <dfn>file extension</dfn> is a string that
+          start with a "." and only contains [=valid suffix code points=]. A
+          <dfn>valid suffix code point</dfn> is a [=code point=] that is
+          [=ASCII alphanumeric=], U+002B (+), or U+002E (.). Additionally,
+          [=file extensions=] are limited to a length of 16 code points.
+        </p>
+        <aside class="note">
+          <p>
+            These code points were chosen to support most pre-existing file
+            formats. The vast majority of file extensions are purely
+            alphanumeric, but compound extensions (such as .tar.gz) and
+            extensions such as .c++ for C++ source code are also fairly common,
+            hence the inclusion of + and . as allowed code points.
+          </p>
+        </aside>
+        <section>
+          <h3>
+            `action` member
+          </h3>
+          <p>
+            The [=file handler=]'s <code><dfn data-dfn-for=
+            "file handler">action</dfn></code> member is a <a>string</a> that
+            represents a URL relative to the <a data-cite=
+            "appmanifest#dfn-manifest-url">manifest URL</a> that is
+            [=manifest/within scope=] of a <a data-cite=
+            "appmanifest#dfn-processed-manifest">processed manifest</a> . This
+            URL will be navigated to in the steps to [=execute a file handler
+            launch=].
+          </p>
+        </section>
+        <section>
+          <h3>
+            `name` member
+          </h3>
+          <p>
+            The [=file handler=]'s <code><dfn data-dfn-for=
+            "file handler">name</dfn></code> member is a <a>string</a> that
+            identifies the file type to the user. User agents MAY pass this
+            information to the operating system during file handler
+            registration.
+          </p>
+          <aside class="note">
+            <p>
+              The [=file handler/name=] is used to identify and describe the
+              file type in a human readable format, and is typically only
+              displayed in OS surfaces (such as a file browser) if the web app
+              has become the default handler for that file type. It's most
+              useful for custom file types.
+            </p>
+          </aside>
+        </section>
+        <section>
+          <h3>
+            `icons` member
+          </h3>
+          <p>
+            The <a>file handler</a>'s <code><dfn data-dfn-for=
+            "file handler">icons</dfn></code> member lists images that serve as
```

Comment:
```suggestion
            "file handler">icons</dfn></code> member lists icons that serve as
```

I think icons is the better word here.

Reply:
done